contracts/StrategyCurve3crv.sol

```diff
@@ -224,43 +291,21 @@ contract StrategyCurve3crv is StrategyCurveBase {
             uint256 _debtPayment
         )
     {
-        // harvest our rewards from the gauge
-        gauge.claim_rewards();
-        uint256 crvBalance = crv.balanceOf(address(this));
-        uint256 gnoBalance = gno.balanceOf(address(this));
-        // if we claimed any CRV, then sell it
-        if (crvBalance > 0) {
-            // keep some of our CRV to increase our boost
-            uint256 sendToVoter = crvBalance.mul(keepCRV).div(FEE_DENOMINATOR);
-            if (keepCRV > 0) {
-                crv.safeTransfer(voter, sendToVoter);
-            }
-
-            // check our balance again after transferring some crv to our voter
-            crvBalance = crv.balanceOf(address(this));
-
-            // sell the rest of our CRV
-            if (crvBalance > 0) {
-                _sellToken(address(crv), crvBalance);
-            }
-        }
-        // sell GNO if we have any
-        if (gnoBalance > 0) {
-            _sellToken(address(gno), gnoBalance);
-        }
-
-        uint256 wxdaiBalance = wxdai.balanceOf(address(this));
-        uint256 usdcBalance = usdc.balanceOf(address(this));
-        uint256 usdtBalance = usdt.balanceOf(address(this));
-
-        // deposit our balance to Curve if we have any
-        if (wxdaiBalance > 0 || usdcBalance > 0 || usdtBalance > 0) {
-            curve.add_liquidity([wxdaiBalance, usdcBalance, usdtBalance], 0);
-        }
-
-        // debtOustanding will only be > 0 in the event of revoking or if we need to rebalance from a withdrawal or lowering the debtRatio
-        uint256 stakedBal = stakedBalance();
+        require(tradeFactory != address(0), "!yswaps");
```

Comment:
you don't longer need this. `tradeFactory`, now `tradeHandler` can be initialized in the constructor.

Reply:
Perfect. I'll remove in the next round if changes.